### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by juanca** [mavenlink/mavenlint#4]
```
Oops! Something went wrong! :(

ESLint couldn't find the plugin "eslint-plugin-jsx-a11y". This can happen for a couple different reasons:

1. If ESLint is installed globally, then make sure eslint-plugin-jsx-a11y is also installed globally. A globally-installed ESLint cannot find a locally-installed plugin.

2. If ESLint is installed locally, then it's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm i eslint-plugin-jsx-a11y@latest --save-dev

If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

**Comment by ahuth** [comment]
Interesting. Thanks for making an issue, @juanca .

On Big Maven I'm seeing it lint JSX files:

```bash
~/workspace/mavenlink (master *) $ yarn lint
yarn lint v0.22.0
$ eslint 'frontend/**/*.js' 'frontend/**/*.jsx' 'app/**/*.js' '.eslintrc.js' 'config/**/*.js' --ignore-pattern '!.eslintrc.js'
The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

/Users/andrew/workspace/mavenlink/frontend/components/error-flag/error-flag.jsx
  1:19  error  Strings must use singlequote  quotes

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

Besides the deprecation warning, it _**seems*__ like it's working. Can't see anything really interesting in Big Maven's .eslintrc, either.

Really weird.

**Comment by juanca** [comment]
Yeah, hmmmm.... I'ma need to take some time to dig further. At the moment, I side stepped the issues to get `mavenlink-ui` rolling but I plan on revisiting this.

**Comment by ahuth** [comment]
May be an issue with the version of ESLint and the versions of the dependencies here. When I try to upgrade ESLint and the dependencies, I start getting the errors you were seeing, @juanca
